training_maquetado/scss/tech.scss

```diff
@@ -20,28 +21,42 @@
     
     &,
     &::before {
-      border-radius: 10px;
+      border-radius: 6px;
       box-sizing: border-box;
+      margin: 2% 0;
     }
 }
 
 .tech-icon {
   height: $tech-logo-size;
-  margin: 4px 0;
+  width: $tech-logo-size;
 }
 
 .tech-title, 
 .tech-release, 
 .tech-desc {
   font-size: $text-medium;
-  margin: 4px 0;
   text-align: center;
 }
 
 .tech-release {
   font-weight: bold;
 }
 
+.tech-title {
+  margin: 12px 0;
+}
+
 .tech-desc {
   margin: 12px 0;
 }
+
+.tech-grid {
+  align-content: space-between;
+  display: flex;
+  flex-direction: row;
+  flex-wrap: wrap;
+  justify-content: space-between;
+  margin: 0 auto 80px;
+  width: 55%;
```

Comment:
Using percent is not necessary, with just margin, padding and width you can center the elements